### PR TITLE
Log upload filta analytics event

### PR DESCRIPTION
**Pull request by asenrusev**
Dem skillz

### CODE REVIEW

File: Editor/DevPanel.cs [hunk 1]
```diff
@@ -821,6 +821,7 @@ public class QueueResponse {
                 SetStatusMessage("Upload successful. Processing... (4/5)");
             }
             AssetDatabase.DeleteAsset(VariantTempSave);
+            Backend.Instance.LogAnalyticsEvent("upload_filta");
```
Comment (cdiddy77) [inline Editor/DevPanel.cs:824]: consider sending also the artid of the filta. It would be useful information.

**Review by cdiddy77 — APPROVED** [review]
I think it would be good to send the artid as a param

File: Editor/DevPanel.cs (hunk 1 above)
Comment (gazuntype) [reply Editor/DevPanel.cs:824]: i agree with this.

**Review by gazuntype — APPROVED** [review]
wow. phenomenal c# skills. simply phenomenal.

File: Editor/DevPanel.cs (hunk 1 above)
Comment (asenrusev) [reply Editor/DevPanel.cs]: Updated. Thanks for the suggestion.

**Pull request merged**